.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 sudo: false
 node_js:
-  - 0.10
+  - 4.2
+  - 5.2
```

Comment:
Is it worth just specifying `4` and `5` here so the latest point release of each is used?